cmd/renterd/main.go

```diff
@@ -462,8 +464,12 @@ func main() {
 	}
 	var shutdownFns []shutdownFnEntry
 
-	if cfg.Bus.RemoteAddr != "" && len(cfg.Worker.Remotes) != 0 && !cfg.Autopilot.Enabled {
-		logger.Fatal("remote bus, remote worker, and no autopilot -- nothing to do!")
+	if cfg.Bus.RemoteAddr != "" {
+		if len(cfg.Worker.Remotes) != 0 && !cfg.Autopilot.Enabled {
+			logger.Fatal("remote bus, remote worker, and no autopilot -- nothing to do!")
+		} else if cfg.Worker.ServiceAddress == "" {
```

Comment:
I was on the fence whether we need to add an extra check here whether it's really necessary to require the service address. I figured it's probably fine to have the user configure it twice in the case a user is configuring a distributed node on localhost.

Reply:
This should be fine. But I'd still manually verify a distributed setups to make sure we didn't miss anything with our assumption.

Reply:
So I was planning to follow up this PR with an added section to our documentation that explains how to set up a distributed `renterd`. I verified it myself, and I'm sure artur who opened the issue will be able to tell us whether this solves his issue in k8s setups.